lib/puppetserver/ca/config/puppet.rb

```diff
@@ -77,8 +77,9 @@ def load(cli_overrides: {}, logger:, ca_dir_warn: true)
         end
 
         def default_certname
-          hostname = Facter.value(:hostname)
-          domain = Facter.value(:domain)
+          networking = Facter.value(:networking)
+          hostname = networking[:hostname]
```

Comment:
I'm not sure if that's actually a symbol or if Facter returns a string, but I think it's all symbols.

Reply:
all strings:
```
$ bundle exec irb
irb(main):001:0> require 'facter'
=> true
irb(main):002:0> networking = Facter.value(:networking).keys
=> ["domain", "fqdn", "interfaces", "ip6", "mac", "mtu", "netmask", "ip", "network", "netmask6", "primary", "network6", "scope6", "hostname"]
irb(main):003:0>
```